packages/axelar-wasm-std/src/voting.rs

```diff
@@ -137,6 +140,8 @@ pub trait Poll {
         Self: Sized;
     // returns the cumulated poll result
     fn result(&self) -> PollResult;
+    // returns the consensus for the message at the given index
+    fn consensus(&self, idx: usize) -> Result<bool, Self::E>;
```

Comment:
I don't like this as part of the trait. Polls don't necessarily have messages. Additionally, the name is a bit weird